include/boost/geometry/algorithms/detail/overlay/follow.hpp

```diff
@@ -487,8 +487,7 @@ public :
                     strategy, robust_policy,
                     linear::get(out));
             }
-            else if (FollowIsolatedPoints
-                  && following::is_touching(*it, *iit, entered))
+            else if (BOOST_GEOMETRY_CONDITION(FollowIsolatedPoints && following::is_touching(*it, *iit, entered)))
```

Comment:
Is there a reason why did you decide to put it in one line instead of keeping the old layout? I.e.:
```
else if (BOOST_GEOMETRY_CONDITION(FollowIsolatedPoints)
      && following::is_touching(*it, *iit, entered))
```

Reply:
None. Perhaps just a legacy and paranoid habit to fit a macro invocation in a single line.

I've reverted the formatting.